ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -25,4 +38,55 @@ export const appAssets: AppAssets[] = [
     logo: rStudioLogo,
     icon: rStudioIcon,
   },
+  {
+    appType: UIAppType.CROMWELL,
+    logo: cromwellLogo,
+    icon: cromwellIcon,
+  },
 ];
+
+// TODO replace with better defaults
```

Comment:
These work as a POC for create/delete so I don't think this is blocking.  But we should update this ASAP.

Reply:
Marcus will ask around for better values for this